demo/m1_facilitator/bin/request_deposit_testnet.ts

```diff
@@ -0,0 +1,117 @@
+import fs from 'fs';
+import path from 'path';
+
+import Deposit from '../lib/Deposit';
+
+const inquirer = require('inquirer');
+const Web3 = require('web3');
+
+const { BN } = new Web3().utils;
+
+interface DepositInputInfo {
+  web3EndPoint: string;
+  valueTokenAddress: string;
+  erc20GatewayAddress: string;
+  transactionGasPrice: string;
+  depositorKeystore: string;
+  password: string;
+  amountToDeposit: string;
+  beneficiary: string;
+  gasPrice: string;
+  gasLimit: string;
+}
+
+async function readInput(): Promise<DepositInputInfo> {
+  const filePath = path.join(__dirname, '/', 'depositor.json');
+  const answer = await inquirer.prompt([
+    {
+      type: 'string',
+      name: 'web3EndPoint',
+      default: 'https://rpc.slock.it/goerli',
+      message: 'Enter origin chain(goerli) end point:',
+    },
+    {
+      type: 'string',
+      name: 'valueTokenAddress',
+      default: '0xd426b22f3960d01189a3d548b45a7202489ff4de',
+      message: 'Enter value token address:',
```

Comment:
Why are we using `value token`  term?
Why not `ERC20 token`?

Reply:
I wanted to be specific to the terms we have been using throughout mosaic-1 and that is why I used `value token`.
Thank you for pointing it out now, for the user-doc I can update it to `ERC20 token`.

Reply:
Updated